lib/sprockets/utils/gzip.rb

```diff
@@ -0,0 +1,64 @@
+require 'concurrent'
```

Comment:
Good call, I moved it when I did some refactoring. I'll see about only requiring `future`.